### DISCUSSION TITLE
Is there a way of having it work just with the timer and without mouse movement?

### DISCUSSION THREAD

**Discussion by Gianluca4646** [rcaelers/workrave#470] [Q&A] [answered]
As it says above as sometimes my work doesn't require mouse movement and it also would be helpful with gaming too if this was the case.

**Comment by mhvis** [comment, marked as answer]
Is Reading Mode what you are looking for?

https://workrave.org/docs/menus/reading-mode/

**Reply by Gianluca4646** [reply]
Thanks! I'll try that one out!
